packages/web-twig/src/Resources/components/Toast/Toast.twig

```diff
@@ -0,0 +1,36 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _alignmentX = props.alignmentX | default('center') %}
+{%- set _alignmentY = props.alignmentY | default('bottom') %}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Toast' -%}
+{%- set _queueClassName = _spiritClassPrefix ~ 'Toast__queue' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+
+{%- set _alignmentClasses = [] -%}
+{%- for alignment in [_alignmentX, _alignmentY] -%}
+    {%- if alignment is iterable -%}
+        {%- for breakpoint, bpValue in alignment -%}
```

Comment:
```suggestion
        {%- for breakpoint, breakpointValue in alignment -%}
```

Am I correct in understanding this? Can we please avoid abbreviations in the naming of variables?

Reply:
I agree, it's a copy-pasted code so I preferred being it the same.